docs/reference/slm/getting-started-slm.asciidoc

```diff
@@ -34,15 +42,22 @@ PUT /_snapshot/my_repository
 [[slm-gs-create-policy]]
 ==== Setting up a snapshot policy
 
-Now that we have a repository in place, we can create a policy to automatically
-take snapshots. Policies are written in JSON and will define when to take
-snapshots, what the snapshots should be named, and which indices should be
-included, among other things.  We'll use the <<slm-api-put-policy>> API
-to create the policy.
+Once you have a repository in place, you can define an {slm-init} policy to automate your backups. 
```

Comment:
We kinda jump between backups and snapshots. I'd establish backup = snapshot upfront and just use one.
```suggestion
Once you have a repository in place, you can define a {slm-init} policy to automate your backups. 
```

Reply:
Oops. Bad suggestion on my part. Please ignore.